pom.xml

```diff
@@ -1,22 +1,26 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
     <groupId>com.bmdsofware.dicoogle</groupId>
     <artifactId>ipfs-plugin</artifactId>
     <name>ipfs</name>
     <version>1.0.0-SNAPSHOT</version>
     <packaging>jar</packaging>
-    
+
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 
         <!-- Dicoogle SDK version here -->
-        <dicoogle.version>3.0.0-SNAPSHOT</dicoogle.version>
+        <dicoogle.version>3.0.0</dicoogle.version>
 
```

Comment:
Are you still using 3.0.0? There is a 3.0.2 release :D
Hope another one soon.

Reply:
Right, I focused on using the not snapshot version and forgot about the 3.0.2 :D

Reply:
It will be deployed. 
We deployed for BMD repos only.